mlkem/verify.h

```diff
@@ -11,6 +11,19 @@
 #include "cbmc.h"
 #include "params.h"
 
+/* Static namespacing */
```

Comment:
Can you add a section to the README regarding single-CU builds, and refer to the section in this comment?
I think this is rather confusing for most C developers.

Reply:
I see you added it in a later commit. Do you want to refer to it here?